agent/common.sh

```diff
@@ -7,6 +7,8 @@ export AGENT_STATIC_IP_NODE0_ONLY=${AGENT_STATIC_IP_NODE0_ONLY:-"false"}
 
 export AGENT_USE_ZTP_MANIFESTS=${AGENT_USE_ZTP_MANIFESTS:-"false"}
 
+export PLATFORM_TYPE=${PLATFORM_TYPE:-""}
```

Comment:
The name of the config var seems pretty generic, being a specif agent one I'd suggest `AGENT_PLATFORM_TYPE`. Also, it would need to be documented in config_example.sh